docs/robots/dingo/integration_dingo/integration_dingo.mdx

```diff
@@ -1,10 +1,233 @@
+used for attaching external payloads. The mounting holes come with M5 screws pre-installed.
+You may mount your hardware directly onto the trough cover or you may design and mount
+a new plate to the trough cover and secure it to the trough cover using M5 screws.
+
+#### Mechanical, PACS
+
+<ComponentPacsMechanical />
+
+Refer to the following pages for Dingo brackets that can simplify your integration.
+You can also try different confgurations using our tool at [build.clearpathrobotics.com/husky](https://build.clearpathrobotics.com/dingo)
+
+- [Top Plate](/docs/robots/dingo/integration_dingo/kits/top_plate_dingo)
+
+### Electrical Integration {#dingo-payload-electrical-integration}
+
+Except for bus-powered USB cameras, most payloads have separate leads for power and data.
+
+#### Data Connections
+
+Data connections may be brought through the openings in the trough cover and connected directly to
+the internal computer.
+
+Dingo's internal computer options support USB3 and Ethernet connectivity. In addition,
+the PCIe Gen3 x16 slot may be used to supply a GPU or other attachements, as necessary.
+Additionally, the internal mounting area may be used for an Ethernet switch, when attaching multiple
+Ethernet payloads, or for a PoE power injector as required.
+
+#### Connector Summary
+
+The Dingo MCU board provides a number of connections for power, relays, auxiliary inputs and
+auxiliary outputs. These are summarized in the following table and described in more detail in
+the following sections.
+
+| MCU board label  | MCU board connector | Mating connector | Crimp terminals                                              | Crimping tool    |
+| :--------------- | :------------------ | :--------------- | :----------------------------------------------------------- | :--------------- |
+| PWR1, PWR2       | Molex 0039301040    | Molex 0039012040 | Molex 0507528200 (AWG 20-22) or Molex 0503518100 (AWG 22-28) | Molex 2002184900 |
+| VBAT             | Molex 768250002     | Molex 1700010102 | Molex 0507528200 (AWG 20-22) or Molex 0503518100 (AWG 22-28) | Molex 2002184900 |
+| INPUTS           | TE 3-794682-8       | TE 794617-8      | TE 1-794610-1 (AWG 20-24) or TE 1-794611-1 (AWG 26-30)       | TE 91391-1       |
+| SW_IN            | TE 2-1445094-2      | TE 1445022-2     | TE 1-794610-1 (AWG 20-24) or TE 1-794611-1 (AWG 26-30)       | TE 91391-1       |
+| AUX1, AUX2, AUX3 | TE 3-794682-4       | TE 794617-4      | TE 1-794610-1 (AWG 20-24) or TE 1-794611-1 (AWG 26-30)       | TE 91391-1       |
+| RELAY            | TE 4-794682-0       | TE 1-794616-0    | TE 1-794610-1 (AWG 20-24) or TE 1-794611-1 (AWG 26-30)       | TE 91391-1       |
+| FAN              | Molex 0533750210    | Molex 0511630200 | Molex 0507528200 (AWG 20-22) or Molex 0503518100 (AWG 22-28) | Molex 2002184900 |
+| ESTOP            | Molex 053370610     | Molex 0511630600 | Molex 0507528200 (AWG 20-22) or Molex 0503518100 (AWG 22-28) | Molex 2002184900 |
```

Comment:
Consider changing 'ESTOP' to 'motion-stop'

Reply:
I'll change estop to motion stop in most places for Dingo, but in this particular case ESTOP is the actual label on the PCB